Adding further information regarding `include_units` (#17069)
DataDog/integrations-core: journald/README.md

- `include_user_units`: Includes all user-level units specified.
- `exclude_user_units`: Excludes all user-level units specified.

> **Note**: Use the `*` wildcard in `exclude_units` or `exclude_user_units` to specify a particular Journald log. Further note that `*` will not work with `include_units`, by default if no units for both System and User, and no matches are defined, all journal logs will be collected.

Reviewer: ```suggestion
**Note**: Use the `*` wildcard in `exclude_units` or `exclude_user_units` to specify a particular Journald log. The `*` does not work with `include_units`. By default, if there are no units for neither system nor user, and no matches are defined, all journal logs are collected.
```
